tests/PySys/apt-plugin/apt-list-format/run.py

```diff
@@ -0,0 +1,33 @@
+import pysys
+from pysys.basetest import BaseTest
+
+"""
+Validate apt plugin output format
+"""
+
+
+class AptPluginTest(BaseTest):
+    def execute(self):
+        apt_plugin = "/etc/tedge/sm-plugins/apt"
```

Comment:
For this to work it requires the system to have the plugin installed in this location, would that be presumed that the workflow performs installation?

Reply:
Yes, the plugin has to be installed first using `cargo deb -p tedge_apt_plugin; sudo dpkg -i target/debian/tedge_apt_plugin_0.1.0_amd64.deb`.